src/features/mcda/utils/openMcdaFile.ts

```diff
@@ -10,6 +11,29 @@ const input = (() => {
   return input;
 })();
 
+function createMCDAConfigFromJSON(json: Partial<MCDAConfig>): MCDAConfig {
+  const result: Partial<MCDAConfig> = { ...json };
+  if (!result.version || result.version < 4) {
```

Comment:
Is it ok to use here magic number `4`?

Reply:
Should be fine, the version is literally definted like this in interface right now:
<img width="483" alt="image" src="https://github.com/konturio/disaster-ninja-fe/assets/9570735/16764c13-9274-4547-bd65-ca45a192d530">
This check is pretty arbitraty tbh.

When new version rolls out, we might want to reorganize it better (we may want backwards compatibility and handle files of different versions differently)